rllib/offline/offline_prelearner.py

```diff
@@ -412,7 +412,7 @@ def _map_sample_batch_to_episode(
                 )
                 # Create a `SingleAgentEpisode`.
                 episode = SingleAgentEpisode(
-                    id_=batch[schema[Columns.EPS_ID]][i][0],
+                    id_=str(batch[schema[Columns.EPS_ID]][i][0]),
```

Comment:
nice catch

Reply:
Yeah, well, I got an error running the connector pipeline.